ui_api.cpp

```diff
@@ -271,14 +318,59 @@ static int l_imgHandleImageSize(lua_State* L)
 	return 2;
 }
 
+class ui_luaReader_c {

```

Comment:
Is this just for future changes?  I'm not opposed to including it, but it'd be better if we had something using it right away to show some examples.

Reply:
Example usage added.